src/Views/HardwareView.vala

```diff
@@ -583,6 +583,24 @@ public class About.HardwareView : Gtk.Grid {
 
         return hostname;
     }
+
+    // Format layperson-friendly size string, replacement for GLib.format_size ().
+    // Always return "GB", "TB" etc. even if IEC_UNITS requested, instead
+    // of "GiB", "TiB" etc. for the benefit of average users.
+    private string custom_format_size (uint64 size, bool iec_unit) {
+        uint divisor = iec_unit ? 1024 : 1000;
+
+        string[] units = { _("bytes"), _("KB"), _("MB"), _("GB"), _("TB"), _("PB")};
+
+        int unit_index = 0;
+
+        while ((size / divisor) > 0 && (unit_index < units.length)) {
+            unit_index++;
+            size /= divisor;
+        }
+
+        return "%llu %s".printf (size, units[unit_index]);
```

Comment:
I don't think that this will be great i18n wise

Reply:
Applying code suggested by @lenemter ...